format/index.md

```diff
@@ -1244,8 +1245,7 @@ A server **MUST** respond to `PATCH`, `POST`, and `DELETE` requests to a
 *to-many relationship URL* as described below.
 
 For all request types, the body **MUST** contain a `data` member whose value
-is an object that contains `type` and `id` members, or an array of objects
-that each contain `type` and `id` members.
+is a linkage object or an array of linkage objects.
```

Comment:
Fixed! This is why I should think while updating, rather than blindly find-and-replace-ing :)